staticDocs/Permissions.md

```diff
@@ -0,0 +1,151 @@
+# OneBlink Role Permissions
+
+There are a collections of permissions that are required to use different functions within the OneBlink SDK. These permissions can be set within the [OneBlink Console](console.oneblink.io). If you do not have access, you will need to work with your Administrator to set the permissions that you need.
+Without certain permissions, you will be unable to complete actions with the OneBlink SDK. The required permissions are included on the function and they will be listed below.
+
+## Endpoints and their Required Roles
+
+### Approvals
+
+| Endpoint                                      | Permission | Permission Level |
```

Comment:
Worth calling these `Function` or `Action` instead of `Endpoint`?

Reply:
yep yep, I started using function above but forgot to change it